packages/nuxt/test/nuxt-link.test.ts

```diff
@@ -125,6 +135,21 @@ describe('nuxt-link:propsOrAttributes', () => {
       it('defaults to `null`', () => {
         expect(nuxtLink({ to: 'https://nuxtjs.org' }).props.target).toBe(null)
       })
+
+      it('target="_blank" with baseURL', () => {
+        vi.mocked(useRuntimeConfig).withImplementation(() => {
+          return {
+            app: {
+              baseURL: '/base'
+            }
+          } as any
+        }, () => {
+          expect(nuxtLink({ to: '/to', target: '_blank' }).props.href).toBe('/base/to')
+          expect(nuxtLink({ to: '/base/to', target: '_blank' }).props.href).toBe('/base/to')
+          expect(nuxtLink({ to: 'http://nuxtjs.org/app/about', target: '_blank' }).props.href).toBe('http://nuxtjs.org/app/about')
+          expect(nuxtLink({ to: '//nuxtjs.org/app/about', target: '_blank' }).props.href).toBe('//nuxtjs.org/app/about')
```

Comment:
Maybe it'd be semantically more correct to check for external links NOT to include the baseURL here.